cypress_test/integration_tests/common/desktop_helper.js

```diff
@@ -270,18 +270,41 @@ function deleteImage() {
 		.should('not.exist');
 }
 
-function insertMultipleComment(numberOfComments) {
-	numberOfComments = numberOfComments || 1;
+// we don't need to scroll if toolbar is already scrolled
+// for the mentioned purpose isFirstCall parameter is used
+function insertMultipleComment(docType, numberOfComments = 1, isFirstCall = true) {
```

Comment:
why to have an extra parameter? instead you could just check if the 'scroll-right' is available then do it straight away!?